src/models/token.js

```diff
@@ -0,0 +1,28 @@
+const { Sequelize, DataTypes } = require('sequelize');
+
+const { User } = require('./user');
+const { sequelize } = require('../utils/db.js');
+
+const Token = sequelize.define(
+  'Token',
+  {
+    refreshToken: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    expireAt: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: Sequelize.literal(`NOW() + INTERVAL 30 DAYS`),
```

Comment:
The `defaultValue` for `expireAt` uses a raw SQL expression `NOW() + INTERVAL 30 DAYS`, which may not be supported directly by Sequelize. Consider using Sequelize's built-in functions or hooks to set the default expiration date.